code/modules/mob/mob_helpers.dm

```diff
@@ -531,3 +531,32 @@
 ///Can the mob see reagents inside of containers?
 /mob/proc/can_see_reagents()
 	return stat == DEAD || has_unlimited_silicon_privilege //Dead guys and silicons can always see reagents
+
+//Gets ID card from a mob. If hand_firsts is TRUE hands are checked first, otherwise other slots are prioritized (for subtypes at least).
+/mob/proc/get_idcard(hand_first)
+	if(!length(held_items)) //Early return for mobs without hands.
+		return
+	//Check hands
+	var/obj/item/held_item = get_active_held_item()
+	if(held_item) //Check active hand
+		. = held_item.GetID()
+	if(!.) //If there is no id, check the other hand
+		held_item = get_inactive_held_item()
+		if(held_item)
+			. = held_item.GetID()
```

Comment:
Getters are pretty bad and we shouldn't expand on them.
The proper way would be to retrieve the active hand held item and have the information on whether that is currently acting as an ID through a variable in it, which is modified accordingly as the events change.
That would be a bigger refactor, though, and not necessary here. Just keep it in mind for future changes.

That said, could you put these getters in the `/mob/living` level? We don't need non-living mobs to check for ID, do we?

Reply:
Done.